permissions/DataStorages.permission

```diff
@@ -5,3 +5,9 @@
 # x-sailjail-description = Removable media
 # x-sailjail-translation-key-long-description =
 # x-sailjail-long-description = Use memory cards and USB sticks
+
+### Allow to listen/talk to udisks2
+# Used for mounting, formatting
```

Comment:
I wonder if we should allow this at all. Then again, we need to check that UDisks.permission still anyway, so it's not too critical to do it now.

Reply:
That was what I was thinking most of the day that should some app have the permission to add/format sd/usb or is it completely internal functionality.

Reply:
Getting it off "Base" would already be a massive improvement.

But there was some reason why it got included in there... I do not remember exactly what it might have been, but typical reason for illogically lavish Base permission would be: without it some mw component crashes on unchecked access failure -> changes like this need testing / enough time in devel.

Reply:
But could we limit the scope of `UDisks2` here as well? There are some a bit dangerous D-Bus methods that can be used with this lavish rule, Delete(), Resize() do not sound like our mw apps do really need. Settings is the only one that can manage removable media and it does not yet use sailjail/firejail?

Reply:
Well, making it stricter would of course be better. With assumption like "only settings modifies" it could even be only  enumeration/getter functionality allowed or stte.

Reply:
I went through https://git.sailfishos.org/mer-core/nemo-qml-plugin-systemsettings/blob/master/src/udisks2defines.h and added the more specific interface/method rules with 3d6d780 + the `org.freedesktop.UDisks2.Manager.GetBlockDevices` which is in the code only and what has been complained in the journal if there are no UDisks2 permissions.

Reply:
I did split the permissions to `UDisks` and to `UDisksListen` - the latter of which is included by `RemovableMedia` and by `UDisks`. This way the apps can simply get info on the removable media, as well as other storage devices on the system but not to tamper with them at any level. All destructive, mounting, etc. are left in `UDisks` permission.